packages/unity-interface/BrowserInterface.ts

```diff
@@ -590,6 +590,17 @@ export class BrowserInterface {
     }
   }
 
+  public RequestHeaderForUrl(data: { method: string, url: string}) {
+    const identity = getCurrentIdentity(store.getState())
+    if (!identity) {
```

Comment:
u can avoid writing the same fn twice with a ternary, so if we need to change something in the future its easier 😄 

```ts
const headers: Record<string, string> = identity 
  ? BuilderServerAPIManager.authorize(identity, data.method, data.url)
  : {}
getUnityInstance().SendBuilderCatalogHeaders(headers)

```

Reply:
Changed!